### ISSUE TITLE
Need some sort of "Sorry, no data available at this point" when all endpoints are 4XX error codes

### ISSUE THREAD

**Issue by BobTorgerson** [ua-snap/arctic-eds#361]
For example, the report at: http://localhost:3000/report/51.5000/-146.0000#results has no valid data as it sits in the middle of the ocean, but still returns the TOC as if there is more to see. 

The API is properly returning empty JSON objects, so testing for that condition should allow for us to show an "error" message for such no data areas.

<img width="1257" alt="ArcticEDS No Data" src="https://github.com/ua-snap/arctic-eds/assets/563260/18e58393-7fe3-492a-9632-1d60bea5f024">

**Comment by brucecrevensten** [comment]
ref #351

**Comment by charparr** [comment]
Yeah, this is still valid: https://arcticeds.org/report/51.5000/-146.0000
